ui/plugins/eu.esdihumboldt.hale.ui/src/eu/esdihumboldt/hale/ui/io/instance/InstanceExportWizard.java

```diff
@@ -117,72 +117,84 @@ public InstanceExportWizard() {
 	public boolean performFinish() {
 		boolean success = super.performFinish();
 
-		if (success && validator != null) {
+		if (success && !validators.isEmpty()) {
 			// validate the written output
 
-			// configure validator
-			List<? extends Locatable> schemas = getProvider().getValidationSchemas();
-			validator.setSchemas(schemas.toArray(new Locatable[schemas.size()]));
-			ExportTarget<?> exportTarget = getSelectTargetPage().getExportTarget();
-			if (exportTarget instanceof FileTarget) {
-				String fileName = ((FileTarget<?>) exportTarget).getTargetFileName();
-				LocatableInputSupplier<? extends InputStream> source = new FileIOSupplier(
-						new File(fileName));
-				validator.setSource(source);
-				validator.setContentType(getContentType());
-
-				// XXX configuration pages for validator?
-
-				IOReporter defReport = validator.createReporter();
-
-				// validate and execute provider
-				try {
-					// validate configuration
-					validator.validate();
-					IOReport report = execute(validator, defReport);
-
-					if (report != null) {
-						// add report to report server
-						ReportService repService = PlatformUI.getWorkbench()
-								.getService(ReportService.class);
-						repService.addReport(report);
-						// show message to user
-						if (report.isSuccess()) {
-							// info message
-							log.userInfo(report.getSummary());
-						}
-						else {
-							// error message
-							log.userError(report.getSummary()
-									+ "\nPlease see the report for more details.");
+			for (InstanceValidator validator : validators) {
+				// set schemas
+				List<? extends Locatable> schemas = getProvider().getValidationSchemas();
+				validator.setSchemas(schemas.toArray(new Locatable[schemas.size()]));
+
+				ExportTarget<?> exportTarget = getSelectTargetPage().getExportTarget();
+				if (exportTarget instanceof FileTarget) {
+					String fileName = ((FileTarget<?>) exportTarget).getTargetFileName();
+					LocatableInputSupplier<? extends InputStream> source = new FileIOSupplier(
+							new File(fileName));
+					validator.setSource(source);
+					validator.setContentType(getContentType());
+
+					// XXX configuration pages for validator?
+
+					IOReporter defReport = validator.createReporter();
+
+					// validate and execute provider
+					try {
+						// validate configuration
+						validator.validate();
+						IOReport report = execute(validator, defReport);
+
+						if (report != null) {
+							// add report to report server
+							ReportService repService = PlatformUI.getWorkbench()
+									.getService(ReportService.class);
+							repService.addReport(report);
+							// show message to user
+							if (report.isSuccess()) {
+								// info message
+								log.userInfo(report.getSummary());
+							}
+							else {
+								// error message
+								log.userError(report.getSummary()
```

Comment:
Same as in the comment related to the ValidationJob - IMHO a user message would be more appropriate after all validators completed (instead of for each).